### PR TITLE
Tests/Python: add multipole python test

### PR DESCRIPTION
**Pull request by RTSandberg**
- add Python run script
- add run script to example documentation
- add automated CTest

### CODE REVIEW

File: examples/multipole/README.rst [hunk 1]
```diff
@@ -16,3 +16,9 @@ In this test, the initial and final values of :math:`\sigma_x`, :math:`\sigma_y`
 .. literalinclude:: input_multipole.in
    :language: ini
    :caption: You can copy this file from ``examples/multipole/input_multipole.in``.
+
+This test can also be run as a Python script (``python3 run_chicane.py``):
```
Comment (n01r) [inline examples/multipole/README.rst]:
```suggestion
This test can also be run as a Python script (``python3 run_multipole.py``):
```

**Review by n01r — COMMENTED** [review]
Just a small comment change, then it also looks good, thank you! :)

**Review by cemitch99 — APPROVED** [review]
It looks good me me, thanks.

File: examples/multipole/README.rst (hunk 1 above)
Comment (RTSandberg) [reply examples/multipole/README.rst]: thanks - good catch

File: examples/CMakeLists.txt [hunk 2]
```diff
@@ -229,6 +229,16 @@ add_impactx_test(multipole
     OFF  # not plotting script yet
 )
 
+# Python: Chain of Multipoles Test ############################################################
```
Comment (ax3l) [inline examples/CMakeLists.txt]: Just aligning things, pls don't judge me :sweat_smile: 
```suggestion
# Python: Chain of Multipoles Test #########################################################
```

File: examples/multipole/README.rst [hunk 3]
```diff
@@ -16,3 +16,9 @@ In this test, the initial and final values of :math:`\sigma_x`, :math:`\sigma_y`
 .. literalinclude:: input_multipole.in
    :language: ini
    :caption: You can copy this file from ``examples/multipole/input_multipole.in``.
+
+This test can also be run as a Python script (``python3 run_multipole.py``):
```
Comment (ax3l) [inline examples/multipole/README.rst:20]: Please put Python above the inputs - see FODO example :)

File: examples/multipole/run_multipole.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+distr = distribution.Waterbag(
+    sigmaX = 4.0e-3,
+    sigmaY = 4.0e-3,
+    sigmaT = 1.0e-3,
+    sigmaPx = 3.0e-4,
+    sigmaPy = 3.0e-4,
+    sigmaPt = 2.0e-3,
+    muxpx = 0.0,
+    muypy = 0.0,
+    mutpt = 0.0)
```
Comment (ax3l) [inline examples/multipole/run_multipole.py]: Zero should be the default for these :)
```suggestion
    sigmaPt = 2.0e-3)
```

File: examples/CMakeLists.txt (hunk 2 above)
Comment (RTSandberg) [reply examples/CMakeLists.txt]: ha ha, no problem

File: examples/multipole/README.rst (hunk 3 above)
Comment (RTSandberg) [reply examples/multipole/README.rst:20]: we decided to do this in a separate issue / PR

**Review by ax3l — APPROVED** [review]

**Pull request merged**